samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -764,6 +764,7 @@ class SamzaContainer(
         containerListener.afterStart()
       }
       metrics.containerStartupTime.set((System.nanoTime() - startTime)/1000000)
```

Comment:
Can we use (`container-startup-time`) metric instead to infer if the container has started?

Reply:
we can but its relatively harder to automate since we need to look for changes